packaging/aznfs/RPM/aznfs.spec

```diff
@@ -63,6 +66,11 @@ systemctl daemon-reload
 systemctl enable aznfswatchdog
 systemctl start aznfswatchdog
 
+echo -e "${RED}******************************************************"
+echo -e "Do not uninstall AZNFS in case of existing mounts present." 
+echo -e "It may lead to broken AZNFS package with removed dependencies."
+echo -e "******************************************************${NORMAL}"
+
```

Comment:
I noticed in one of your screen sharing sessions that the red color is not very readable.
also put an echo before and after this

echo
echo "*******************************************************************"
echo "Do not uninstall AZNFS while you have active aznfs mounts!"
echo "Doing so may lead to broken AZNFS package with unmet dependencies."
echo "If you want to uninstall AZNFS make sure you unmount all aznfs mounts."
echo "********************************************************************"
echo